### ISSUE TITLE
License

### ISSUE THREAD

**Issue by uXeBoy** [kghunt/NWatch#2]
Still missing the original license?

https://github.com/zkemble/NWatch/blob/master/License.txt

**Comment by kghunt** [comment]
Yes I have done anything with the nwatch code at all. I only cloned it so I
could upload the other guys code. I will put the license back in when I
clone it on my new laptop.

I can add you as a collaborator if you wish?
